### DISCUSSION TITLE
How to : Mermaid support in vitepress .md files

### DISCUSSION THREAD

**Discussion by ezzle** [mermaid-js/mermaid#4511] [Q&A]
Hi,

I am struggling with mermaid & vitepress integration.  The [vitepress-plugin-mermaid](https://github.com/emersonbottero/vitepress-plugin-mermaid) does not work in my project. I already open an unanswered issue.

As your doc uses vitepress can you help with an How-to. I can see that diagrams are correctly rendered in your doc.

I tried to to explore your repo in github but I do not figure out the required configuration, logic  and work flow.

Thank you in advance for your help.

Best regards,

Tshitshi

**Comment by sidharthv96** [comment]
Hi @ezzle,
Please have a look inside [packages/mermaid/src/docs/.vitepress](https://github.com/mermaid-js/mermaid/blob/develop/packages/mermaid/src/docs/.vitepress) to see how we've integrated.
